pages/AdListPage/[id]/index.js

```diff
@@ -98,7 +112,6 @@ const Description = styled.section`
 `;
 
 const Attributes = styled.section`
-  align-self: flex-start;
   padding: 0 1rem 1rem 1rem;
 
   p {
```

Comment:
I strongly recommend to avoid nested styling. As your styles won't be really reusable this way.